pkg/apis/cincinnati/v1beta1/cincinnati_types.go

```diff
@@ -25,6 +25,9 @@ type CincinnatiSpec struct {
 	// GraphDataImage is a container image that contains the Cincinnati graph
 	// data. The data is copied to /var/lib/cincinnati/graph-data.
 	GraphDataImage string `json:"graphDataImage"`
+
+	// Address defines which address should be used when services bind to the network
```

Comment:
We could do a bool for ipv6 and based on it's value, set the address to `0.0.0.0` or `::/0`. WDYT?

Reply:
That was my first approach. But then since you're using a template for building the configmap I didn't know how to translate that to the template. If you provide the guidance I'll go that way.

Reply:
Added IPv6 boolean to the Spec.